### PR TITLE
[integration test] add a new profile test

### PR DESCRIPTION
**Pull request by kongtiaowang**
## Brief summary of changes
testing add a new profile.
#### Testing instructions (if applicable)
passing git actions.

### CODE REVIEW

**Comment by cmadjar** [comment]
@ycAbout any chance you could review this?

**Comment by ycAbout** [comment]
Sure! @cmadjar

File: modules/new_profile/test/new_profileTest.php [hunk 1]
```diff
@@ -79,36 +67,35 @@ function testNewProfilePageLoads()
      */
     function testNewProfileCreateCandidate(): void
     {
-        $this->markTestSkipped(
-            'Skipping tests until Travis and React get along better'
-        );
         $this->setUpConfigSetting("useEDC", "false");
```
Comment (ycAbout) [inline modules/new_profile/test/new_profileTest.php:70]: Should we also add test for the saturation that useEDC is set to true?

**Review by ycAbout — COMMENTED** [review]
Great. Looks all good to me. Just one little suggestion in the comment!

File: modules/new_profile/test/new_profileTest.php (hunk 1 above)
Comment (kongtiaowang) [reply modules/new_profile/test/new_profileTest.php:70]: This function only tests the "Create Candidate" function. 
If this test works fine with git action, I will add the useEDC in the future.

**Comment by kongtiaowang** [comment]
replaced

**Pull request closed without merging**